app/Providers/AppServiceProvider.php

```diff
@@ -125,8 +125,22 @@ public function boot()
                     return '';
                 }
 
+
+                if(! Utils::isNinjaProd() && \App::isLocale('en') == false) {
```

Comment:
And this one...

Reply:
Handles fixing a mismatched breadcrumb when the module name is different.  The reason it specifically looks for the non-English locales is there is another check at line 142 that handles the English locale using the built-in str_singular function.

That being said, I'm wondering if the issue lies deeper inside the module translation code; I might keep looking a bit at it to see if I can fix the issue more elegantly.

Reply:
I'm happy to help try to find a better solution, the purpose of the code is a bit hard to understand.